js/components/portal-dashboard/response-details/response-details.tsx

```diff
@@ -55,24 +57,25 @@ export class ResponseDetails extends React.PureComponent<IProps, IState> {
   }
   render() {
     const { activities, anonymous, answers, currentActivity, currentStudentId, currentQuestion, hasTeacherEdition, isAnonymous, questions,
-      setAnonymous, setCurrentActivity, setCurrentStudent, setStudentFilter, sortByMethod, sortedQuestionIds, studentCount, students,
+      setAnonymous, setCurrentActivity, setCurrentQuestion, setCurrentStudent, setStudentFilter, sortByMethod, sortedQuestionIds, studentCount, students,
       trackEvent } = this.props;
     const { selectedStudents, showSpotlightDialog, showSpotlightListDialog, inQuestionMode } = this.state;
     // TODO: FEEDBACK
     // if feedback is on, show the QuestionFeedbackPanel or the Activity FeedbackPanel
     const firstActivity = activities.first();
     const firstQuestion = questions?.first();
     const currentStudentIndex = students.findIndex((s: any) => s.get("id") === currentStudentId);
+    const sequence = activities.size > 1;
```

Comment:
Might consider renaming this variable.  As named, it implies that it holds a sequence object, but it's really used to tell if a sequence exists (correct?).  Maybe consider naming it hasSequence or sequenceExists or something a little more clear.

Reply:
Or even multipleActivities.  Anyway, something to give a better clue to its use below.